### ISSUE TITLE
Display TPoS transaction details

### ISSUE THREAD

**Issue by AlexITC** [wiringbits/block-explorer#72]
When displaying a TPoS contract transaction, its details aren't displayed

### Expected behavior

When viewing a TPoS contract transaction, the merchant, owner and commission are displayed.

### Actual behavior

Loading a TPoS contract transaction doesn't display its details.

### Steps to reproduce the behavior

Go to: https://xsnexplorer.io/transactions/9f874ccf1ec091788ca1d26e22d1c22376af635e0d2272a822646f8de9e0eb87

**Comment by bwang22** [comment]
Is this problem when the details show when searching by blockhash

**Comment by AlexITC** [comment]
No, a contract is embedded in a transaction, we could display the contract info while displaying a transaction.
